pom.xml

```diff
@@ -265,7 +265,7 @@
 			<dependency>
 				<groupId>org.hibernate</groupId>
 				<artifactId>hibernate-validator</artifactId>
-				<version>5.4.0.Final</version>
+				<version>6.1.5.Final</version>
```

Comment:
@ibacher shouldn't the validator so be using the `hibernateVersion` property?
 So currently the validator is on an older version than the other hibernate deps since hibernate version is 5.4.18.Final